### DISCUSSION TITLE
Generating ARM64 compatible OSquery Package with FleetDM?

### DISCUSSION THREAD

**Discussion by Edelweiss1996** [Security-Onion-Solutions/securityonion#8679] [General] [answered]
I am currently looking to expand OSquery onto several new servers that are running on ARM64 Architecture.
The packages on the 'download' tab of the SOC don't specify the Architecture of the package, though I assume it is x86_64. Looking at FleetDM itself, I don't see a way to specify the package that fleet will generate, only -type which is relegated to pkg/msi/deb/rpm.

This might not be the correct place to ask, but is there a way through Security Onion and it's fleet integration to produce a package that works on ARM64? Otherwise, should the standard package it generates on the download page work or will I need to look into manually enrolling a standard version of OSquery into FleetDM?

**Comment by reyesj2** [comment, marked as answer]
No, there is not a way to generate the ARM64 package from Fleet. I think you'll be best off looking into enrolling a plain osquery into FleetDM best support would come from [FleetDM. ](https://fleetdm.com/docs/using-fleet/adding-hosts#plain-osquery)

**Reply by kgoode517** [reply]
I am also actively looking into this. Currently talking with them on their slack. https://osquery.slack.com/archives/C01DXJL16D8/p1662250629292309

**Reply by Edelweiss1996** [reply]
Hi there, thank you for the response. I did expect this but I wanted to exhaust all my options and see if the Sec Onion version of Fleet (and Fleet in general) might have additional packages to generate. It looks like for now I'll be working with OSquery's Arm64 package and manually enrolling it into Fleet which is a bit outside the scope of Security Onion's discussion.
Thank you for the quick answer and pointing me in the right direction, it helps a lot.
